meta/documents/user_guide/en/settings_guide.md

```diff
@@ -129,26 +129,48 @@ When `Send all inventory items to Wayfair?` is disabled, selling an Item on Wayf
 
 Enabling `Send all inventory items to Wayfair?` will allow Wayfair to list any item that is determined to be currently in stock, including those Item Variations for which "Wayfair" is **not** present in the `Markets` list.
 
+## Inventory page
+**(Since 1.1.4)**
 
-## Full Inventory page
-The `Full Inventory` page does not contain any settings to configure.
-It is used for checking the status of syncing the Plentymarkets inventory with Wayfair, or manually initiating a synchronization of all inventory items.
-The page's data will automatically refresh over time, but you may also manually refresh it.
+The 'Inventory' page does not contain any settings to configure.
+It is used for checking the status of syncing the Plentymarkets inventory with Wayfair.
+The page's data refreshes on a regular basis, as denoted by the timestamp toward the bottom.
+
+### Main synchronization status
+
+The top of the `Inventory` page always displays an icon and message to describe the overall health of the inventory synchronization process.
+
+* A calendar icon ![calendar](../../../images/common/icon_calendar.png) will appear if the Wayfair plugin has yet to attempt inventory synchronizations.
+
+    * This happens right after the plugin is installed.
+    * This may happen if the plugin was recently upgraded.
+    * The page will change within 15-30 minutes, as the synchronization system initializes.
+
+* An icon with a checkmark ![checked_cloud](../../../images/common/icon_cloud_check.png) will appear if the Wayfair plugin is regularly synchronizing inventory without detecting any issues.
 
-**The Wayfair plugin periodically sends inventory updates to Wayfair, without any further manual activations. The Full Inventory page only displays information about the daily updates that include all items in the inventory.**
+* An icon with a crossed-out cloud ![crossed_cloud](../../../images/common/icon_cloud_slash.png) will appear if the Wayfair plugin detects irregularities in inventory synchronization.
 
-### Fields
-* The `Time of last successful inventory synchronization` fields keep track of the daily synchronizations that are normally preformed automatically.
-    * A "check mark" icon ![cloud with check](../../../images/common/icon_cloud_check.png) will appear if it has been less than 24 hours since the last successful synchronization. This indicates that no actions are required.
-    * A "warning" icon ![warning icon](../../../images/common/icon_warning.png) will appear if it has been over 24 hours since the last successful synchronization. Consider using the `Start Synchronization` button as described below.
+    * Errors are expected if the Wayfair plugin was recently installed and [the setup process](initial_setup.md) has not been completed yet. Finish setting up the plugin prior to using the `Inventory` page again.

```

Comment:
Do we want to call this out at the top of the document as well? This may help ensure they don't start seeing errors and reaching out.

Reply:
Maybe not the top of _this_ document in particular, but I agree that we can call it out somewhere that the supplier is likely to see it early in the process of installation / setup.